core/pom.xml

```diff
@@ -19,6 +19,46 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-webflux</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.5</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>ua.test.mega</groupId>
+			<artifactId>mega-tester-adapters</artifactId>
+			<version>0.0.1-SNAPSHOT</version>
```

Comment:
Better to use ${project.version} and specify it in the dependency manager in main pom.

Reply:
done